src/coresyftools/coresyftools/tests/test_gpt_tool.py

```diff
@@ -99,11 +102,11 @@ def __call__(self, args):
         tool.execute(cmd)
 
         gpt_cmd = 'gpt {} -f GeoTIFF-BigTIFF -t {} -param=val {}'.format(
-            os.path.join(os.getcwd(), 'gpt_graph.xml'),
+            os.path.join(os.getcwd(), graph_file_name),
             os.path.join(os.getcwd(), 'output'), os.path.join(os.getcwd(), 'input')).split()
         self.assertEqual(call_shell_command_mock.args, gpt_cmd)
         os.remove('input')
-        os.remove('gpt_graph.xml')
+        os.remove(graph_file_name)
```

Comment:
Include these steps into "TearDown" method.

Reply:
It would fail, because only this test creates this file (remember that `tearDown` is run after each test).